NanoProd/python/customize.py

```diff
@@ -79,6 +86,20 @@ def customizePV(process):
 
   return process
 
+def addSpinnerWeights(process):
+  process.tauSpinnerWeightTable = cms.EDProducer(
+    "TauSpinnerTableProducer",
+    branch   = cms.string("TauSpinner"),
+    input    = cms.InputTag("prunedGenParticles"),
+    #theta    = cms.string("0,0.25,0.5,-0.25,0.375"),
+    theta    = cms.vdouble(0,0.25,0.5,-0.25,0.375),
+    pdfSet   = cms.string("NNPDF30_nlo_as_0118"),
```

Comment:
Do we want to use the same PDF set the same for Run2 UL and Run3? Is this the most up-to-date value?

Reply:
It is what I do not know and it is why I made it configurable. I guess that the best will be use same PDF set as used to produce Higgs samples. On the other hand, I suppose that this has negligible impact of the final result. Anyway, to be followed at CP HTT forum.

Reply:
After some thinking: I am not entirely sure but it can be that choice of a PDF set can be not important for CP of the tau-H-tau coupling for which weights are computed by this module. It is important for tau polarization in Z decays as it imcts Z production.

Reply:
If it is not that important, I propose using one of the "preferred" NNLO PDFs from [genWeightsTable_cfi.py]( https://github.com/cms-sw/cmssw/blob/babb58fca451d00a49fce98204870f36aea9d4e8/PhysicsTools/NanoAOD/python/genWeightsTable_cfi.py#L8-L13). So that we close the PR and I start reprocessing of HTT CP samples. Would you like to confirm it first in the CP HTT mattermost?

Reply:
It seems that neither PDF set not sqrt(s) are important for Higgs CP. Nevertheless, sqrt(s) is adjusted to data taking era and a modern PDF set used.